src/domain/safe/entities/operation.entity.ts

```diff
@@ -1,6 +1,4 @@
-export const CALL_OPERATION = 0;
-export const DELEGATE_OPERATION = 1;
-
-export type Call = typeof CALL_OPERATION;
-export type Delegate = typeof DELEGATE_OPERATION;
-export type Operation = Call | Delegate;
+export enum Operation {
+  CALL = 0,
+  DELEGATE = 1,
+}
```

Comment:
This is used by more entities than the `PreviewTransactionDto`. I would isolate this change first.

Reply:
I reverted the simplification in 1ebbadfa4a0de17ec5417785c91460b244969784.